fenix/app/src/main/java/org/mozilla/fenix/autofill/AutofillUtils.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.autofill
+
+import android.util.Log
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+import mozilla.components.concept.storage.LoginEntry
+import mozilla.components.service.sync.logins.LoginsApiException
+import mozilla.components.service.sync.logins.SyncableLoginsStorage
+import org.mozilla.fenix.browser.BaseBrowserFragment
+import kotlin.coroutines.cancellation.CancellationException
+
+internal fun BaseBrowserFragment.handleOnSaveLoginWithGeneratedStrongPassword(
```

Comment:
Should we just put this function as a private function in BaseBrowserFragment?  Or is there a reason that we need it as an extension?